web-api/src/business/useCaseHelper/trialSessions/generateNoticeOfChangeToInPersonProceeding.ts

```diff
@@ -68,7 +67,7 @@ export const generateNoticeOfChangeToInPersonProceeding = async (
       data: {
         caseCaptionExtension,
         caseTitle,
-        docketNumberWithSuffix,
+        docketNumberWithSuffix: docketNumberWithSuffix || docketNumber,
```

Comment:
TODO: Let's make this a computed property. `docketNumberWithSuffix` should always be defined and just return `Case.getDocketNumberWithSuffix`. The `||` here is just a hack to get the types right.

Reply:
In fact, I am just now realizing that we set the `docketNumberWithSuffix` in different ways that we should standardize. In `Case.ts`, despite having `docketNumberSuffix` in the DB, we always recompute it [here](https://github.com/ustaxcourt/ef-cms/blob/b9cbce59f8841c5ddb02f9ab133995883d422556/shared/src/business/entities/cases/Case.ts#L761). Elsewhere, we sometimes rely on that field being in the DB and then stitch it on.

Maybe `docketNumberWithSuffix` should just be a computed field that always calls `getDocketNumberSuffix`; if so, we just store `docketNumber` and `caseType` on the DB.

Reply:
I've looked into this a little. It isn't quite as straightforward as I had hoped, but I'll work on this in a separate branch.